src/CMakeLists.txt

```diff
@@ -590,12 +590,8 @@ if(USE_CURL)
 	include_directories(${CURL_INCLUDE_DIR})
 endif()
 
-
-# When cross-compiling assume the user doesn't want to run the executable anyway,
-# otherwise place it in <source dir>/bin/ since Luanti can only run from there.
-if(NOT CMAKE_CROSSCOMPILING)
-	set(EXECUTABLE_OUTPUT_PATH "${CMAKE_SOURCE_DIR}/bin")
-endif()
```

Comment:
this is just removing the logic that was there before and not a proper fix.
instead the path could be set to `${CMAKE_BINARY_DIR}`